src/utilities/OpenWeather.test.ts

```diff
@@ -0,0 +1,76 @@
+import axios from "axios";
+import getTemperature from './OpenWeather';
+
+jest.mock("axios");
+
+const error = {data:
+  {
+    "cod": '404',
+    "message": "city not found"
+  }
+}
+
+const data = {data: 
+  {     
+    "coord": {
+      "lon": -87.7603,
+      "lat": 41.9699
+    },
+    "weather": [
+      {
+        "id": 803,
+        "main": "Clouds",
+        "description": "broken clouds",
+        "icon": "04d"
+      }
+    ],
+    "base": "stations",
+    "main": {
+      "temp": 55,
+      "feels_like": 48.07,
+      "temp_min": 48.99,
+      "temp_max": 53.6,
+      "pressure": 1020,
+      "humidity": 43
+    },
+    "visibility": 10000,
+    "wind": {
+      "speed": 9.22,
+      "deg": 90
+    },
+    "clouds": {
+      "all": 75
+    },
+    "dt": 1620497789,
+    "sys": {
+      "type": 1,
+      "id": 5228,
+      "country": "US",
+      "sunrise": 1620470276,
+      "sunset": 1620521827
+    },
+    "timezone": -18000,
+    "id": 0,
+    "name": "Chicago",
+    "cod": 200
+  }}
+
+describe('test getTemperature', () => {
+  afterEach(() => jest.resetAllMocks());
+
+  it("returns temp", async () => {
+    axios.get.mockResolvedValue(data);
+    const temp = await getTemperature('60630');
+    expect(axios.get).toHaveBeenCalled();
+    expect(temp).toEqual('55')
+    expect(axios.get).toBeCalledWith("http://api.openweathermap.org/data/2.5/weather", {"params": {"appid": "fb7c51ef434a235b3397829e4e8d9ce7", "units": "imperial", "zip": "60630"}});
```

Comment:
Instead of hard coding the `appid`, what do you think about adding `process.env.REACT_APP_OPENWEATHER_API_KEY`. Right now the tests fail for me since I need to set that env var. Later on, we should have a config file to store the environment variables we will use and import them to the required files. I'll make a ticket for that.

Reply:
Fixed. This was an oversight, good catch.